video/out/opengl/context_android.c

```diff
@@ -122,18 +136,14 @@ static bool android_init(struct ra_ctx *ctx)
 
 static bool android_reconfig(struct ra_ctx *ctx)
 {
-    struct priv *p = ctx->priv;
-    int w, h;
+    void *tmp = talloc_new(NULL);
+    struct android_opts *opts = mp_get_config_group(tmp, ctx->global, &android_conf);
 
-    if (!eglQuerySurface(p->egl_display, p->egl_surface, EGL_WIDTH, &w) ||
-        !eglQuerySurface(p->egl_display, p->egl_surface, EGL_HEIGHT, &h)) {
-        MP_FATAL(ctx, "Failed to get height and width!\n");
-        return false;
-    }
```

Comment:
I think we should fallback to the egl width/height when the user doesn't provide any values.